docs/content/dev-straat.md

```diff
@@ -0,0 +1,95 @@
+# Dev straat
+
+Eisen:
+
+* Server om referentie-implementatie op te hosten
+* Automatische deployment master branch referentie-implementatie
+* Tooling om spec json/yaml file te genereren uit referentie-implementatie
+* Automatisch committen van spec update bij mergen van user story
+* ReDoc -> github pages als eerste opzet, auto-deploy via Travis
```

Comment:
Is Jenkins ook OK?

Reply:
Prima! Maakt Maykin niets uit (we draaien intern ook Jenkins)

Reply:
Ik vind Travis wel mooier: https://travis-ci.org/Haarlem/zaakregistratiecomponent
Hoeven we het ook niet zelf te beheren en is een beetje standaard voor open source projecten. Aan de andere kant is Jenkins wss iets flexibeler.